--- dbus-glib-0.98/dbus/dbus-gobject.c.format-string	2011-10-01 12:54:49.409256372 +0200
+++ dbus-glib-0.98/dbus/dbus-gobject.c	2011-10-01 12:54:35.068252606 +0200
@@ -40,7 +40,7 @@
 G_GNUC_NORETURN static void
 oom (const gchar *explanation)
 {
-  g_error (explanation == NULL ? "Out of memory" : explanation);
+  g_error (explanation == NULL ? "Out of memory" : "%s", explanation);
   g_assert_not_reached ();
 }
 
